mysql-test/suite/innodb/r/mem_pressure.result

```diff
@@ -0,0 +1,23 @@
+#
+# MDEV-24670 avoid OOM by linux kernel co-operative memory management
+#
+CREATE TABLE t1 (t TEXT) ENGINE=InnoDB;
+INSERT INTO t1 SELECT CONCAT(REPEAT('junk text', 500), CAST(seq AS CHAR)) FROM seq_1_to_1000;
+SELECT CAST(VARIABLE_VALUE AS INTEGER) INTO @dirty_prev FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME='Innodb_buffer_pool_pages_dirty';
+SELECT CAST(VARIABLE_VALUE AS INTEGER) INTO @flush_prev FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME='Innodb_buffer_pool_pages_flushed';
+set debug_dbug="d,trigger_garbage_collection";
+SET GLOBAL innodb_buffer_pool_size=@@innodb_buffer_pool_size;
+SELECT SLEEP(5);
+SLEEP(5)
+0
+SELECT VARIABLE_VALUE < @dirty_prev AS LESS_DIRTY_IS_GOOD FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME='Innodb_buffer_pool_pages_dirty';
+LESS_DIRTY_IS_GOOD
+1
+SELECT VARIABLE_VALUE > @flush_prev AS MORE_FLUSHED_IS_GOOD FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME='Innodb_buffer_pool_pages_flushed';
+MORE_FLUSHED_IS_GOOD
+1
```

Comment:
The `buf_pool::garbage_collect()` avoids any writes in order to minimize the I/O impact. If something was written out during the 5-second sleep, that should be a coincidence. I’d rather omit this part of the test.